api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -87,111 +88,154 @@
   private static final String CDR_VERSION_BIGQUERY_DATASET = "CDR_VERSION_BIGQUERY_DATASET";
   private static final String CONFIG_FILENAME = "config/all_of_us_config.json";
 
+  private static final Workspace constructListWorkspaceFromDb(org.pmiops.workbench.db.model.Workspace workspace,
```

Comment:
You might want to add a comment saying what's *not* populated in the list responses.